src/czbenchmarks/metrics/implementations.py

```diff
@@ -146,3 +159,43 @@
     default_params={"metric": "auroc", "classifier": None},
     tags={"label_prediction"},
 )
+
+metrics_registry.register(
+    MetricType.ACCURACY,
```

Comment:
We're going to get conflicts on these because I also had to add these: https://github.com/chanzuckerberg/cz-benchmarks/pull/380/files#diff-b4751e066d5363c5911ffa5ac2e71bef33cb4ee56bb48fdcd4e4a72478b44f72R116

But I implemented them by having them computed within the task, similar to how we're doing the mean-over-folds metrics, rather than requiring the task to return a full set of predictions